xclim/indicators/atmos/_conversion.py

```diff
@@ -382,3 +384,28 @@ def cfcheck(self, **das):
     var_name="mrt",
     compute=indices.mean_radiant_temperature,
 )
+
+
+shortwave_radiation_upwards_from_net_downwards = Converter(
+    title="Upwards shortwave radiation",
+    identifier="shortwave_radiation_upwards_from_net_downward",
+    units="W m-2",
+    standard_name="surface_net_upward_shortwave_flux",
+    long_name="Upwards shortwave radiative flux",
```

Comment:
```suggestion
    standard_name="surface_upwelling_shortwave_flux",
    long_name="Upwelling shortwave flux",
```
As per the cf standard names table.